### DISCUSSION TITLE
Kovalardaki sular sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#269] [Zeka Sorusu]
Elimizde dolu bir 16 litrelik kova, boş bir adet 10 litrelik kova ve yine boş bir 6 litrelik kova vardır. Dolu kovadaki suyu kullanarak 8 litrelik su elde etmek isteniyor. Nasıl yaparız?

**Comment by Alperg123456** [comment]
```
16     0     0
0     10     6
6     10     0
6     4     6
12     4     0
12     0     4
2     10     4
2     8     6 
```
